--- src/vdr-plugin-pin/Makefile.orig	2014-03-28 19:57:05.686959321 +0100
+++ src/vdr-plugin-pin/Makefile	2014-03-28 19:57:20.422911531 +0100
@@ -89,7 +89,7 @@
 %.mo: %.po
 	msgfmt -c -o $@ $<
 
-$(I18Npot): $(wildcard *.c)
+$(I18Npot): $(wildcard *.c *.h)
 	xgettext -C -cTRANSLATORS --no-wrap --no-location -k -ktr -ktrNOOP --package-name=vdr-$(PLUGIN) --package-version=$(VERSION) --msgid-bugs-address='<see README>' -o $@ `ls $^`
 
 %.po: $(I18Npot)
